README.md

```diff
@@ -89,7 +89,12 @@ The following examples illustrate the CLI usage.
 If you want to contribute to this project you can create a fork and a pull request. The project is built with Gradle, so you can import it as Gradle project within Eclipse or IntelliJ IDEA (or any other IDE supporting Gradle).
 
 ```bash
+sdk env install
```

Comment:
Maybe put a pointer to sdkman here? Not everyone uses or knows the tool... Maybe say that this is optional. One just has to make sure JDK >17 is installed.